sites/cheerpx/src/content/docs/20-faq.md

```diff
@@ -76,3 +76,9 @@ For more information on how to setup a HTTP server, please see our [server setup
 [server setup guide]: /docs/guides/nginx
 [SharedArrayBuffer]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer
 [cross-origin isolated]: https://web.dev/articles/why-coop-coep
+
+## Do I need an Exit Node for WebVM networking?
+
+You do not need to create an exit node for WebVM. WebVM relies on tailscale to handle networking, allowing acces to your local or development machine as part of the Tailscale network.
```

Comment:
We need to clarify that you don't need an exit node _if_ you only need access to other machines on the tailscale network (including other WebVMs and the local development machine.